spec/factories/entities.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :entity do
+    name { Faker::Company.name }
+    amount { Faker::Number.decimal(l_digits: 2) }
```

Comment:
- [OPTIONAL] Excellent writing unit tests. 👍🏻 Now, let's make sure to also add integration tests with capybara for all the most important components of the RoR application. ✍🏻

![image](https://user-images.githubusercontent.com/79058364/221361585-894123ad-7e2c-4bb3-939a-7106066cc8c6.png)